### DISCUSSION TITLE
payment

### DISCUSSION THREAD

**Discussion by hizlivecilgin** [Chia-Network/chia-blockchain#6908] [Support]
I did not receive payment from chia blockchain uden. I won 2 chia. The first day I won 1.8, I sent it to my gate.io address. I sent the remaining 0.2 chia years when I saw that 24 hours had passed but it had not passed. Transferred in 10 minutes. please help. I have been waiting for payment for 1 week, 1.8 chia does not appear in my transaction history... I sent the same chia to my gate.io chia wallet in both transactions, only 0.2 chia arrived
![3](https://user-images.githubusercontent.com/86287774/122884458-dcc4f980-d346-11eb-9393-21f5ae851af7.jpg)
![2](https://user-images.githubusercontent.com/86287774/122884464-dd5d9000-d346-11eb-98e8-4d8aff4d4108.jpg)
![Screenshot_1](https://user-images.githubusercontent.com/86287774/122884466-ddf62680-d346-11eb-99b9-698987cdb33d.png)
![1](https://user-images.githubusercontent.com/86287774/122884468-de8ebd00-d346-11eb-90e8-b1f377d72692.jpg)
![4](https://user-images.githubusercontent.com/86287774/122884986-55c45100-d347-11eb-9777-b07a545dfa18.png)

**Comment by loppefaaret** [comment]
please contact gate.io, both transaction are showing on xchscan.com:
https://xchscan.com/address/xch1wgw67849gjurvvtn3kk9hnk7nfz6hrcx87kxvmneddckrj682xjqy8mlpx

**Reply by hizlivecilgin** [reply]
it doesn't appear in the image below and currently 1.8 chia is not showing in my processes

**Reply by loppefaaret** [reply]
perhaps not, but that address is holding a total of 2 XCH, from 2 transactions, one 1.8 XCH on block 440301, and a 0.2 on block 445363
if that is your deposit address from gate.io - gate.io _should_ credit your account ?

**Reply by loppefaaret** [reply]
from Chia's perspective, your wallet has send off those 2 transactions to the above address, and that address is now controlling the funds - wether or not the receiver wishes to credit your account or not is a question for them to answer

**Reply by hizlivecilgin** [reply]
hımm oky thank you

**Reply by hizlivecilgin** [reply]
I have one more question. It makes agriculture simultaneously with spare and plax applications. Do you think it's okay for us to do this?

**Reply by hizlivecilgin** [reply]
spare blockchain . flax blackchain

**Reply by loppefaaret** [reply]
I'm a Chia employee - all i can say, is that be careful with your Chia private keys! - if you use the same plots and keys on their chain - make sure you got a secure way to farm:
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management
it is a 3rd party to Chia, and from that perspective, protect your self!
but from a technical perspective, it should be possible to co-farm different chains with the same plots

**Reply by computer817** [reply]
Hello, you see again, I also didn't arrive! GATE. IO cannot arrive, the official will help us communicate and coordinate with the GATE to solve this problem!

**Comment by BrandtH22** [comment]
@hizlivecilgin , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
